docs/_data/jekyll_filters.yml

```diff
@@ -18,16 +18,19 @@
 #
 - name: Relative URL
   description: >-
-    Prepend the <code>baseurl</code> value to the input. Useful if
-    your site is hosted at a subpath rather than the root of the domain.
+    Transforms a file path into the corresponding relative url by prepending the 
+    <code>baseurl</code> value to the input. Useful if your site is hosted at a subpath 
+    rather than the root of the domain.
   examples:
     - input: '{{ "/assets/style.css" | relative_url }}'
       output: '/my-baseurl/assets/style.css'
 
 #
 
 - name: Absolute URL
-  description: Prepend the <code>url</code> and <code>baseurl</code> value to the input.
+  description: >-
+    Transforms a file path into the corresponding relative url by prepend 
```

Comment:
you mean absolute URL here?

Reply:
This seems kind of verbose to say `transforms` and `corresponding`.

Rather:

> Convert a URL to an absolute URL by prepending the <code>url</code> and <code>baseurl</code> values to the input.

Also, the filter name itself (`absolute_url`) is explanatory enough to say it converts a URL to an absolute URL. So you could just leave this description was it was originally.